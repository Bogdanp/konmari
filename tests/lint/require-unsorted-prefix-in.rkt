#lang racket/base

(require racket/string
         (prefix-in f: racket/format)
         racket/file
         "b.rkt"
         "a.rkt")
