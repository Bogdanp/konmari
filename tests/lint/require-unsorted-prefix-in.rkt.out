/require-unsorted-prefix-in.rkt:4:10:warning:require "racket/format" should come before "racket/string"
/require-unsorted-prefix-in.rkt:5:10:warning:require "racket/file" should come before "racket/format"
/require-unsorted-prefix-in.rkt:7:10:warning:require "a.rkt" should come before "b.rkt"
